src/LanguageServer/Impl/LanguageServer.cs

```diff
@@ -99,7 +97,8 @@ private class PublishDiagnosticsParams {
         }
 
         #region Events
-        private void OnTelemetry(object sender, TelemetryEventArgs e) => _telemetry.SendTelemetry(e.value);
+        [Obsolete]
+        private void OnTelemetry(object sender, TelemetryEventArgs e) { }
```

Comment:
You can simply remove it - it is private.

Reply:
Yep